packages/squiggle-lang/__tests__/TS/Score_test.ts

```diff
@@ -0,0 +1,19 @@
+import { testRun } from "./TestHelpers";
+
+describe("KL divergence", () => {
```

Comment:
Just call "Log Score"; that's what we call it elsewhere.

Reply:
No, these are different things

Reply:
Maybe, but I think that "log score" is inherently ambiguous. But if you search for the KL divergence, you will instantly find the concept that the code is instantiating

Reply:
If that's the case though, we'd need to standardize this, and call it "KLDivergence" throughout the codebase, instead of just this specific test case instance

Reply:
Makes sense